packages/core/src/lib/TableNew/TableNew.stories.tsx

```diff
@@ -0,0 +1,224 @@
+import * as React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import {
+    header,
+    syncHeader,
+    data,
+    syncData,
+    pageSize,
+    maxPages,
+    customTableMockData,
+} from './MockData';
+import { Loading } from '../Loading';
+import { color } from '@web3uikit/styles';
+import styled from 'styled-components';
+import { Skeleton } from '../Skeleton';
+
+import TableNew from './TableNew';
+export default {
+    title: 'Layout/TableNew',
```

Comment:
This creates a new section that is not needed.
Please change this to `3.Layout/TableNew`